md/release-notes.md

```diff
@@ -35,27 +53,50 @@ Extract informations from a project sources to generate an asciidoc document. Th
 #### Increase the default value of the development runtime's maximum memory heap size to 1Gb (512Mo previously)
 To avoid memory issues with large bar files deployment the default xmx value has been increased to 1Gb. If you want to revert this change, you can change it in the Studio preferences -> Server settings -> Tomcat Maximum memory allocation.
 
-#### UIPath connector: Cloud support
+#### UIPath connector - Cloud support
 The UIPath connector now supports the cloud solution.
 
 #### Collaboration - Select the branches to clone
-When cloning a project, gain time by selecting the branches you want to clone instead cloning all the branches.
+When cloning a project, gain time by selecting the branches you want to clone instead of cloning all the branches.
 
 #### Description field on widgets
 Add documentation to your pages by providing information on a widget (e.g. how to use it), thanks to the new Description field. Use this information to [generate documentation](release-notes.md#project-documentation).
+ 
+#### Bonita Platform Extensibility: Connectors and Actor filters archetypes
+You can now develop and test custom connectors and actor filters independently of the Bonita Studio.
+Bonita offers maven archetypes to help bootstrap these extension points
 
+More information:
+* [Bonita actor filter archetype](actor-filter-archetype.md)
+* [Bonita connector archetype](connector-archetype.md)
 
-### Runtime
+::: info
+This is not limited to 7.11 and can be used from version 7.7
+:::
```

Comment:
Has this been added to documentation of versions 7.7 to 7.10?

Reply:
Yes